distributed/deploy/ssh.py

```diff
@@ -295,14 +295,14 @@ def SSHCluster(
     >>> client = Client(cluster)
 
     An example using a different worker module, in particular the
-    ``dask-cuda-worker`` command from the ``dask-cuda`` project.
+    ``CUDAWorker`` worker from the ``dask-cuda`` project.
 
     >>> from dask.distributed import Client, SSHCluster
     >>> cluster = SSHCluster(
     ...     ["localhost", "hostwithgpus", "anothergpuhost"],
     ...     connect_options={"known_hosts": None},
     ...     scheduler_options={"port": 0, "dashboard_address": ":8797"},
-    ...     worker_module='dask_cuda.dask_cuda_worker')
+    ...     worker_module="dask_cuda.CUDAWorker")
```

Comment:
```suggestion
    ...     worker_module="dask_cuda.cli.dask_cuda_worker")
```

Reply:
`dask_cuda.CUDAWorker` is the class that creates the `Nanny` object. I think what's expected here is the CLI interface, which is what I suggested above.

Reply:
Thanks @pentschev -- I thought this was a `worker_class=` keyword which is more commonly used than `worker_module=`. Updating now

Reply:
Yeah, it can feel a bit counterintuitive as in most cases we're working with Python without a CLI, but to launch over SSH this takes a pretty straightforward path of just launching the command remotely.